.website/.vitepress/theme/style.css

```
@@ -102,36 +175,68 @@
+}
+
+.tag {
+	font-family: 'Space Mono', 'JetBrains Mono', monospace;
+	display: inline-block;
+	border-width: 1px;
+	border-color: currentColor;
+	padding-left: 0.75rem;
+	padding-right: 0.75rem;
+	padding-top: 0.25rem;
+	padding-bottom: 0.25rem;
+	font-family: Space Mono, JetBrains Mono, monospace;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: Mono

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)